setup.cfg

```diff
@@ -4,3 +4,5 @@ test=pytest
 [tool:pytest]
 # Ignore thousands of tests in dependencies installed in a virtual environment
 norecursedirs = lib lib64
+# At some point probably want to make this "strict"
+asyncio_mode = auto
```

Comment:
This is new to me! I'll be happy to remove our [hacks to do this](https://github.com/jupyterhub/jupyterhub/blob/39da98f13345fa1bee1cfae34bf9bf7d748ab438/jupyterhub/tests/conftest.py#L60-L70).

I think auto mode is what we want, and is [recommended by pytest-asyncio](https://pypi.org/project/pytest-asyncio/#auto-mode). We don't have a reason to switch to strict, which is only recommended for testing with multiple async frameworks, which isn't relevant here.